app/jobs/unzip_job.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'zip'
+
+# Change work version from a zipfile type to a browser type by unzipping the attached zip file.
+class UnzipJob < BaseDepositJob
+  queue_as :default
+
+  def perform(work_version)
+    zip_attached_file = work_version.attached_files.last
+    destroy_attached_files_except(zip_attached_file, work_version)
```

Comment:
I could see making a case for this method living on the `WorkVersion` but I don't feel strongly about that.

Reply:
(i.e., I could also see leaving it right here.)